CMSgov/easi-app
# [EASI-2762] Include documents when seeding TRB requests

# EASI-2762

## Changes and Description

- Added documents to the seed cases when seeding TRB requests

## How to test this change

run `/scripts/dev db:seed` and head to http://localhost:3000/trb to check the seeded TRB requests

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
